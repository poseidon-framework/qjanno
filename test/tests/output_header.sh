qhs -H -O "SELECT foo,baz FROM basic.csv"
