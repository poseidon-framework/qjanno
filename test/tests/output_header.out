foo baz
a0 a2
b0 b2
c0 c2
